src/ms-graph/__tests__/__recordings__/fetchOrganization_3927641460/recording.har

```diff
@@ -0,0 +1,305 @@
+{
+  "log": {
+    "_recordingName": "fetchOrganization",
+    "creator": {
+      "comment": "persister:JupiterOneIntegationFSPersister",
+      "name": "Polly.JS",
+      "version": "6.0.5"
+    },
+    "entries": [
+      {
+        "_id": "b403352d03ca320f7aafd080b7496b1b",
+        "_order": 0,
+        "cache": {},
+        "request": {
+          "bodySize": 202,
+          "cookies": [],
+          "headers": [
+            {
+              "name": "accept",
+              "value": "application/json"
+            },
+            {
+              "name": "content-type",
+              "value": "application/x-www-form-urlencoded"
+            },
+            {
+              "name": "accept-encoding",
+              "value": "gzip,deflate"
+            },
+            {
+              "name": "user-agent",
+              "value": "azsdk-js-identity/1.5.2 core-rest-pipeline/1.9.0 Node/v14.15.0 OS/(x64-Linux-5.14.0-1036-oem)"
+            },
+            {
+              "name": "x-ms-client-request-id",
+              "value": "aa96fed3-39d4-4da4-8e7a-07ddd25c649f"
+            },
+            {
+              "name": "content-length",
+              "value": "202"
+            },
+            {
+              "name": "host",
+              "value": "login.microsoftonline.com"
+            }
+          ],
+          "headersSize": 435,
+          "httpVersion": "HTTP/1.1",
+          "method": "POST",
+          "postData": {
+            "mimeType": "application/x-www-form-urlencoded",
+            "params": [],
+            "text": "response_type=token&grant_type=client_credentials&client_id=a8626c1e-191d-4e8f-9cbc-a4a6e85104ac&client_secret=L3X8Q~BO57QkoAsXMSkevrmVR2qiNh.qKEuzucAt&scope=https%3A%2F%2Fgraph.microsoft.com%2F.default"
```

Comment:
It is important to ensure that no credentials are recorded in any commit to the repo. See https://github.com/JupiterOne/graph-microsoft-endpoint-defender/blob/8b834b13321bfbaaed184d1f506cf726d131c5da/test/recording.ts for the place to manage this, with https://github.com/JupiterOne/graph-microsoft-365/blob/a2cceb9734bf9715ccbc87a53c0a5e1fbc8dc96f/test/recording.ts providing example code.